docs/docs/how-to/local-development/prettier.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Using Prettier With Gatsby
+---
+
+Prettier is an opinionated code formatter that auto-formats files commonly used with Gatsby. This makes your code more readable without long lists of stylistic comments polluting PR reviews, or, worse, team conflict over the right way to format code.
+
+Prettier formats JSX, CSS and variants (including Sass and styled-components), and Markdown (including MDX), all of which are commonly used on Gatsby sites.
+
+## Get started with Prettier
+
+There are two different ways to install Prettier:
+
+### Method 1: Set up with new project
+
+All official Gatsby starters come with Prettier installed. Start a new project by installing any official Gatsby starter.
+
+```sh
+gatsby new my-default-starter https://github.com/gatsbyjs/gatsby-starter-default
```

Comment:
should we start recommending the new `npm init gatsby`?

Reply:
Yes